proto/interchain_security/ccv/provider/v1/tx.proto

```diff
@@ -88,4 +89,24 @@ message MsgOptOut {
   string provider_addr = 2 [ (gogoproto.moretags) = "yaml:\"address\"" ];
 }
 
-message MsgOptOutResponse {}
\ No newline at end of file
+message MsgOptOutResponse {}
+
+// MsgSetConsumerCommissionRate allows validators to set 
+// a per-consumer chain commission rate
+message MsgSetConsumerCommissionRate {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+  // The chain id of the consumer chain to set a commission rate
+  string chain_id = 1;
```

Comment:
nit: change the order between chain_id and provider_addr. Usually the first field is the signer.